template/e2e/init.js

```diff
@@ -0,0 +1,25 @@
+const detox = require('detox');
+const config = require('../package.json').detox;
+const adapter = require('detox/runners/jest/adapter');
+const specReporter = require('detox/runners/jest/specReporter');
+
+// Set the default timeout
+jest.setTimeout(120000);
```

Comment:
Should we break this out into a constant and give it a meaningful name? I think it would make it easier to read

Reply:
Thanks, @jsjoeio that totally makes sense! This is the default that gets created with `detox init` but I const it out and put it in the setup and agree I think it makes it a little more clear/readable.